src/routes/api/auth/index.ts

```diff
@@ -43,6 +42,47 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
       return { message: 'Invalid username or password.' }
     }
   )
-}
+  fastify.post(
+    '/update',
```

Comment:
This endpoint is strange, `update` should be expressed by `method`.

Reply:
Yep, we should use `PUT` if we allow to update the resource entirely or `PATCH` if we apply partial changes.